docs/pages/Getting_Started.md

```diff
@@ -2332,24 +2332,24 @@ observation-specific code for reading, writing, or initializing the
 observation, then the entire contents of the new file is:
 
 ~~~
-! BEGIN DART PREPROCESS KIND LIST
+! BEGIN DART PREPROCESS QUANTITY LIST
```

Comment:
check 'quantity' vs 'kind' in this doc.

Reply:
No longer correct

````
Please see the list of available observation quantities in
`assimilation_code/modules/observations/DEFAULT_obs_kind_mod.f90` to determine	`assimilation_code/modules/observations/DEFAULT_obs_kind_mod.f90`
````

Reply:
finding and fixing old lines in code will require grep rather than execution-time tests, because right now preprocess accepts both these lines as synonyms for backwards compatibility.  but we definitely should be documenting the QUANTITY version.

Reply:
documentation change